client/signup/steps/user/index.jsx

```diff
@@ -450,7 +467,7 @@ export class UserStep extends Component {
 			return translate( 'Let’s get you signed up.' );
 		}
 
-		if ( ! headerText ) {
+		if ( isSocialFirst && ! headerText ) {
```

Comment:
Is there a reason `isSocialFirst` shouldn't have a header fallback?

Reply:
Updated as I think it should always have the same header when social first